package.json

```diff
@@ -14,7 +14,8 @@
   },
   "dependencies": {
     "hoek": "2.x.x",
-    "boom": "2.x.x"
+    "boom": "2.x.x",
+    "json-stringify-safe": "5.x.x"
```

Comment:
@hueniverse Just curious, why ? I'm trying to make the new maintainer move on to a new (working) version.